src/gameobjects/ufo.js

```diff
@@ -13,7 +15,8 @@ class UFO extends GameObject {
     this.startRot = -300;
     this.physics = new Physics(new Vector3(0, 0, this.startSpeed), new Vector3(0, this.startRot, 0), 0);
     this.scale = 0.5;
-    this.initalDia = 99.7135610685365;
+    this.extents = manager.modelList.ufo.extents;
+    this.initalDia = this.extents.dia;
```

Comment:
Isnt this redundant?

Reply:
Actually, `this.initalDia` isn't even changed, only referenced, so I can just rename it to `dia`